src/callbacks/callbacks.jl

```diff
@@ -259,6 +259,7 @@ function save_to_disk_func(integrator)
             w_velocity = Geometry.WVector.(ᶜu),
             temperature = TD.air_temperature.(thermo_params, ᶜts),
             potential_temperature = TD.dry_pottemp.(thermo_params, ᶜts),
+            specific_enthalpy = TD.specific_enthalpy.(thermo_params, ᶜts),
```

Comment:
This is probably increasing overall allocations but, if we’re lucky, not in the flame graph allocation test.